### DISCUSSION TITLE
[VoiceOver] Unable to select day in datepicker control

### DISCUSSION THREAD

**Discussion by BakktThavasukkannu** [microsoft/fluentui#26560] [General] [answered]
Describe the issue:
When voice over is enabled in Iphone, user is unable to select a day since focus is coming to select only on month but not the day.

Actual behavior:
Focus is not coming to day.

Expected behavior:
Focus should be on the day so that user can select day.

#11273 -  Is this issues got resolved ?

**Comment by micahgodbolt** [comment, marked as answer]
discussions are not the place to file bugs.
